Common/GPU/Vulkan/VulkanRenderManager.cpp

```diff
@@ -22,6 +22,47 @@
 
 using namespace PPSSPP_VK;
 
+bool VKRGraphicsPipeline::Create(VulkanContext *vulkan) {
+	if (!desc) {
+		// Already failed to create this one.
+		return false;
+	}
+	VkPipeline pipeline;
+	VkResult result = vkCreateGraphicsPipelines(vulkan->GetDevice(), desc->pipelineCache, 1, &desc->pipe, nullptr, &pipeline);
+	this->pipeline = pipeline;
+	delete desc;
+	desc = nullptr;
+	if (result == VK_INCOMPLETE) {
+		// Bad return value seen on Adreno in Burnout :(  Try to ignore?
+		// Create a placeholder to avoid creating over and over if something is broken.
+		pipeline = VK_NULL_HANDLE;
```

Comment:
Hm, it might be nice to log the shader if we can?  I guess it's farther away now.

-[Unknown]

Reply:
Added a comment for now..